migrations/schema.sql

```diff
@@ -91,7 +91,8 @@ CREATE TABLE public.projects (
     issues_count integer NOT NULL,
     tags character varying[] NOT NULL,
     created_at timestamp without time zone NOT NULL,
-    updated_at timestamp without time zone NOT NULL
+    updated_at timestamp without time zone NOT NULL,
+    github_updated_at timestamp without time zone
```

Comment:
This is correct but the schema.sql is overwritten by the migrations, could you add a new migration (you could use the `buffalo  db generate add_github_updated_at_to_issues` to generate a migration) and add the new column through sql? 
You could see an example of this approach here https://github.com/ossn/fixme_backend/blob/master/migrations/20180830180819_AvoidNullRepoTags.up.fizz